modules/microshift-install-rhel-image-mode-run-container.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-run-container_{context}"]
+= Running the {microshift-short} bootc container
+
+If the host is already configured to have a `rhel` volume group with free space, this configuration is inherited by the container. It can be used by the {microshift-short} LVMS Container Storage Interface (CSI) to allocate storage.
+
+.Prerequisites
+
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have SUDO permissions configured.
+* You have a pull secret file for downloading the required {microshift-short} container images.
+* You configure host container storage for reusing available container images.
```

Comment:
```suggestion
* You configured a host container storage for reusing available container images.
```

Reply:
Not sure if it should be "configure"  here or not.

Reply:
Yep, should be past tense, we did that already.